cmd/aws-actuator/main.go

```diff
@@ -120,13 +120,16 @@ func createCommand() *cobra.Command {
 				cmd.Flag("userdata").Value.String(),
 			)
 			if err != nil {
-				return err
+				return fmt.Errorf("unable to create read resources: %v", err)
 			}
 
-			actuator := createActuator(machine, awsCredentials, userData)
+			actuator, err := createActuator(machine, awsCredentials, userData)
+			if err != nil {
+				return fmt.Errorf("unable to create actuator: %v", err)
+			}
 			result, err := actuator.CreateMachine(cluster, machine)
 			if err != nil {
-				return err
+				return fmt.Errorf("unable to create machine: %v", err)
 			}
 			fmt.Printf("Machine creation was successful! InstanceID: %s\n", *result.InstanceId)
```

Comment:
This looks like debug; delete or move to logging?

Reply:
Replace all relevant `fmt.Printf` with `glog.Infof`.